datalad/support/exceptions.py

```diff
@@ -0,0 +1,57 @@
+# emacs: -*- mode: python; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 noet:
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the datalad package for the
+#   copyright and license terms.
+#
+# ## ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+""" datalad exceptions
+"""
+
+
+class CommandError(RuntimeError):
```

Comment:
Well, the way it is now, that wouldn't be correct, since it's meaning is more general. But we could introduce more detailed exceptions derived from that one. For example CommandNotAvailableError is a derived one and could be thrown even before trying to call the command, so the more verbose name of it's parent class would be somewhat misleading.
Anyway, let's do this in a later PR, since this one's cross dependency with PR #90 due to necessary refactoring is annoying already.